### ISSUE TITLE
Color-contrast: Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by maxneuvians** [cds-snc/notification-api#98]
```
<span class="cds-tag alpha">Alpha</span>
```

Element has insufficient color contrast of 3.96 (foreground color: #ffffff, background color: #f90277, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

- Speak to Eman what colours we should use? Or Julianna?

**Comment by timarney** [comment]
Fixed with https://github.com/cds-snc/notification-api/issues/99
